common/eth2/src/lighthouse.rs

```diff
@@ -361,19 +352,6 @@ impl BeaconNodeHttpClient {
         self.get_opt::<(), _>(path).await.map(|opt| opt.is_some())
     }
 
-    /// `GET lighthouse/database/info`
```

Comment:
My feeling is that we shouldn't remove this function even if it's unused in Lighthouse. It feels strange to have an incomplete interface for the `/lighthouse` API. If the test is the issue, then I would argue to remove it only and leave the actual function. Perhaps the main checks in the test can be moved directly into the `store` or `http_api` crates somehow.

I'll also note that if we do decide to remove the function, I believe this would _technically_ count as a breaking change

Reply:
Yeah these are fair points, perhaps if the day comes where the `eth2` crate is publishable we can revisit these missing functions.